src/database/schemas/task.ts

```diff
@@ -0,0 +1,43 @@
+import mongoose from 'mongoose';
+import ITask from '../../models/task';
+
+const taskSchema = new mongoose.Schema<ITask>({
+  title: {
+    required: true,
+    type: String
+  },
+  status: {
+    required: true,
+    type: Number
+  },
+  createdAt: {
+    required: true,
+    type: Date
+  },
+  sortId: {
+    required: true,
+    type: Number
+  },
+  editedAt: {
+    required: false,
+    type: Date
+  },
+  deletedAt: {
+    required: false,
+    type: Date
+  },
+  writing: {
+    required: false,
+    type: String
+  },
+  parentId: {
+    required: false,
```

Comment:
is it supposed for a task to have a linked structure like a tree or linked list?

Reply:
Child will have parentId this way I could build the tree of the tasks